tests/src/test/javascript/crypto/EccCryptoContextSuite.js

```diff
@@ -0,0 +1,245 @@
+/**
```

Comment:
Are there any test vectors we could add to both the Java and JavaScript unit tests?